source/utilities.tex

```diff
@@ -5805,9 +5805,7 @@
 \begin{itemdescr}
 \pnum
 \effects
-Constructs an object of class
-\tcode{bitset<N>},
-initializing all bits to zero.
+Initializes all bits to zero.
```

Comment:
Could we say "all bits in `*this`" here? Without the implied context from "Constructs a meow of class woof," this suggests that it initializes ALL BITS EVERYWHERE IN THE UNIVERSE.

Reply:
Why does my computer still work, then?

Fixed.